src/app/shared/post-it-collections/post-it-collections.component.ts

```diff
@@ -162,8 +163,8 @@ export class PostItCollectionsComponent implements OnInit, OnDestroy {
     this.collectionService.createCollection(postIt.collection)
       .pipe(finalize(() => postIt.initializing = false))
       .subscribe(
-        code => {
-          postIt.collection.code = code;
+        collection => {
+          postIt.collection.code = collection.code;
           postIt.initialized = true;
           this.getCollection(postIt);
```

Comment:
A `TODO` comment could be handy, to know that once the collection returned from `createCollection` is a valid collection from server, that no 'refreshing' to sync witch server is needed.

Reply:
We are going to rewrite data manipulation in those components using this NgRx store. They will not use the services directly anymore so these comments are not necessary in my opinion. What do you think?

Reply:
If all the `someService.subscribe()`s will be rewritten to use NgRx, then the comments won't be needed. If not, it would be nice to have a reminder about parts of code that need to be removed.